sdk/spring/spring-cloud-azure-feature-management/src/main/java/com/azure/spring/cloud/feature/manager/FeatureManager.java

```diff
@@ -57,9 +58,22 @@ public class FeatureManager {
      * @return state of the feature
      * @throws FilterNotFoundException file not found
      */
-    public Mono<Boolean> isEnabledAsync(String feature) throws FilterNotFoundException {
+    public Mono<Boolean> isEnabledAsync(String feature) {
         return Mono.just(checkFeatures(feature));
     }
+    
+    /**
+     * Checks to see if the feature is enabled. If enabled it check each filter, once a single filter returns true it
+     * returns true. If no filter returns true, it returns false. If there are no filters, it returns true. If feature
+     * isn't found it returns false.
+     *
+     * @param feature Feature being checked.
+     * @return state of the feature
+     * @throws FilterNotFoundException file not found
+     */
+    public Boolean isEnabled(String feature) throws FilterNotFoundException {
```

Comment:
When will this method throw `FilterNotFoundException`? Is it better to return `true/false` when `FilterNotFoundException` been throwed?

Reply:
All filters are defined as Spring Beans, if the filter isn't found then we throw an error as they should be defined. This logic is the same across our Spring and DotNet libraries.

Reply:
Got it. Thanks.